test/jasmine/tests/isosurface_test.js

```diff
@@ -374,56 +364,33 @@ describe('Test isosurface', function() {
                     ].join('\n')
                 });
             })
-            .then(delay(20))
-            .then(_hover3)
-            .then(function() {
```

Comment:
Why is this block gone?

Reply:
It does not do anything positive other than causing errors. For the 3rd and 4th point locations we get different positions when using CI.